src/pages/therapist.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { withTranslation } from "next-i18next";
+import Button from "@/components/ui/Button";
+import Input from "@/components/ui/Input";
+import Image from "next/image";
+import PageIntro from "@/components/PageIntro";
+function Therapist({ t }) {
+    return (
+        <div className='g-8 flex flex-wrap items-center lg:justify-start xl:justify-start md:justify-start mx-12 my-8 space-x-2 justify-start'>
```

Comment:
I think ALL these classes are not affect at all.
wrap with container class to adding padding in both sides.
That means, replace all these classes with container class, and adding a py-20

Reply:
it was like this and you said remove the linestyling🌝